modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -1346,9 +1358,9 @@ private ObjectField _updateObjectField(
 		}
 
 		if (Validator.isNotNull(newObjectField.getRelationshipType())) {
-			_validateObjectRelationshipDeletionType(objectFieldId, required);
+			//_validateObjectRelationshipDeletionType(objectFieldId, required);
```

Comment:
Are those comments suposed to still be here?

Reply:
No, I forgot to remove them. I just did now